adapter/internal/oasparser/envoyconf/access_loggers.go

```diff
@@ -36,6 +36,63 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+// getAccessLogConfigs provides file access log configurations for envoy
+func getInsightsAccessLogConfigs() *config_access_logv3.AccessLog {
+	var logFormat *file_accesslogv3.FileAccessLog_LogFormat
+	logpath := defaultAccessLogPath //default access log path
+
+	logConf := config.ReadLogConfigs()
+
+	if !logConf.InsightsLogs.Enable {
+		return nil
+	}
+
+	formatters := []*corev3.TypedExtensionConfig{
+		{
+			Name: "envoy.formatter.req_without_query",
+			TypedConfig: &anypb.Any{
+				TypeUrl: "type.googleapis.com/envoy.extensions.formatter.req_without_query.v3.ReqWithoutQuery",
+			},
+		},
+	}
+	// Set the default log format
+	logFormat = &file_accesslogv3.FileAccessLog_LogFormat{
+		LogFormat: &corev3.SubstitutionFormatString{
+			Format: &corev3.SubstitutionFormatString_TextFormatSource{
```

Comment:
Don't we need this to be JSON format? It is there below. Would be ideal if we could reuse the same code in both places when generating this logFormat.